inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -57,10 +57,60 @@ std::string translate_type_name(const std::string& name) {
     return name;
 }
 
+size_t hash_combine(const void* v, int64_t size) {
```

Comment:
Please add comments to this method.
Why do you use 0x9e3779b9 constant. If I understand right it is because you implement Golden Ratio Hashing.
But it will be better if we have comments about it in the code.